application/dash_application/utility/df_manipulation.py

```diff
@@ -370,11 +371,12 @@ def get_rnaseqqc2_merged():
 def get_pinery_samples(active_projects_only=True):
```

Comment:
The optional `active_projects_only` is never overridden. I think it's cleaner to just remove this parameter and have the environmental variable be the decider. However, I can totally see a future where one Dashi view needs to have all projects and all the other ones don't.

Reply:
Yes, that very possible future was why I left it in as well :grimacing: